src/js/tools/sentryBabelTransformer.ts

```diff
@@ -0,0 +1,43 @@
+import componentAnnotatePlugin from '@sentry/babel-plugin-component-annotate';
+
+import { enableLogger } from './enableLogger';
+import { loadDefaultBabelTransformer } from './sentryBabelTransformerUtils';
+import type { BabelTransformer, BabelTransformerArgs } from './vendor/metro/metroBabelTransformer';
+
+enableLogger();
```

Comment:
Do we for sure want this call here?

Reply:
It only enables it when `env.SENTRY_LOG_LEVEL === 'debug'`. The babelTransformer is created when the file loads, so I call it here not to miss any logs (if enabled).